modules/javafx.controls/src/test/java/test/javafx/scene/control/ComboSpecialKeyTest.java

```diff
@@ -0,0 +1,153 @@
+/*
```

Comment:
This file needs to have a standard copyright header. Can be found in other test files.